awx/api/serializers.py

```diff
@@ -4480,7 +4480,10 @@ def __init__(self, *args, **kwargs):
                 body = messages[event].get('body', {})
                 if body:
                     try:
-                        potential_body = json.loads(body)
+                        rendered_body = (
+                            sandbox.ImmutableSandboxedEnvironment(undefined=DescriptiveUndefined).from_string(body).render(JobNotificationMixin.context_stub())
+                        )
+                        potential_body = json.loads(rendered_body)
```

Comment:
looks like this got copied over from another branch

Reply:
Oh no!! Thank you so much for spotting that!

Reply:
@AlanCoding I believe I have successfully deleted the change to serializers.py that was added by mistake.